specs/FIFO18KX2.yaml

```diff
@@ -177,12 +177,12 @@ ports:
 
 parameters:
     DATA_WRITE_WIDTH1:
-      desc: FIFO data write width, FIFO 1 (1-18)
+      desc: FIFO data write width, FIFO 1 (9, 18)
       type: integer
       default: 18
       range: 1 .. 18
```

Comment:
Why not define the range to be a comma seprated list of valid numbers. eg. range: [9, 18]?
is it do able?